src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -457,15 +457,43 @@ export default {
             this.objectViewMap = {};
             this.layoutItems.forEach(this.trackItem);
         },
+        isItemAlreadyTracked(child) {
+            let found = false,
+                keyString = this.openmct.objects.makeKeyString(child.identifier);
+
+            this.layoutItems.forEach(item => {
+                if (item.identifier) {
+                    let itemKeyString = this.openmct.objects.makeKeyString(item.identifier);
+
+                    if (itemKeyString === keyString) {
```

Comment:
Could you use areIdsEqual from objectApi here? May cut out a few steps.

Reply:
Yes, it does the same thing. I am the one that added that api method, but keep forgetting about it. This works for now though